nerfstudio/data/dataparsers/blender_dataparser.py

```diff
@@ -42,6 +43,11 @@ class BlenderDataParserConfig(DataParserConfig):
     """How much to scale the camera origins by."""
     alpha_color: str = "white"
     """alpha color of background"""
+    ply_path: Path = Path("sparse_pc.ply")
+    """Path to the pointcloud file relative to the data path."""
+    load_3D_points: bool = True
```

Comment:
can we replace these two fields with just one:

```python
ply_path: Optional[Path] = None
```

?

and indicate in the docstring that this is used for Gaussian splatting initialization?